src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import core.basesyntax.models.Color;
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
+        return String.valueOf(Color.values()[index]);
```

Comment:
[Use name() for getting String representation of enum constants ](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#use-name-for-getting-string-representation-of-enum-constants)